src/services/mongo.js

```diff
@@ -14,8 +14,16 @@ const connectToDB = () => {
     useUnifiedTopology: true,
   };
 
+  if (typeof process.env.MONGO_URI === 'undefined') {
+    logger.fatal('Connection unsuccessful: the URI was not provided');
+  }
```

Comment:
```suggestion
  if (typeof process.env.MONGO_URI === 'undefined') {
    logger.fatal('Connection unsuccessful: the URI was not provided');
    return;
  }
```

Reply:
We should throw an error so that the caller can handle it and know it failed

Reply:
> We should throw an error so that the caller can handle it and know it failed

True, updated my comment